test/Microsoft.ApplicationInsights.AspNetCore.Tests/TelemetryInitializers/DomainNameRoleInstanceTelemetryInitializerTests.cs

```diff
@@ -17,7 +17,7 @@ public class DomainNameRoleInstanceTelemetryInitializerTests
         [Fact]
         public void InitializeThrowIfHttpContextAccessorIsNull()
         {
-            Assert.Throws<ArgumentNullException>(() =>
+           var ex = Assert.ThrowsAny<ArgumentNullException>(() =>
```

Comment:
Why grab a reference to the exception if it is not then used later?  I'm guessing it is an exception due to the lack of readability from using var.